contracts/staking/PoSValidatorManager.sol

```diff
@@ -27,8 +34,20 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
         uint256 _maximumStakeAmount;
         /// @notice The minimum amount of time a validator must be staked for.
         uint64 _minimumStakeDuration;
+        /// @notice The minimum delegation fee required to delegate to a validator.
+        uint256 _minimumDelegationFee;
```

Comment:
Delegation fee on the p-chain is a percentage of the staking award. Is minimumDelegationFee also a percentage, or is it a fixed fee?

Reply:
updated to take in bips, meant to be a percentage